great-idea/css/index.css

```diff
@@ -63,4 +63,235 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+/* Your code starts here! */
+/* General Styles */
+p{
+	font-family: 'Nunito', sans-serif;
+	line-height: 1.25;
+}
+a {
+	text-decoration: none;
+}
+ol{
+	list-style: square;
+	list-style-position: inside;
+}
+h1{
+	font-size: 5em;
+	margin-bottom: 35px; 
+}
+nav{
+	justify-content: space-between;
+	width: 100%;
+	height: 120px;
+	align-items: center;
+	margin-right: 100px;
+}
+nav a{
+	
+	color:black;
+}
+footer {
+	text-align: center;
+	margin: 40px auto 120px;
+}
+.container{
+	width: 85%;
+	margin: 0 auto;
+}
+.flex{
+	display: flex;
+}
+.border-top{
+	border-top: 1px solid black;
+}
+.border-bottom{
+	border-bottom: 1px solid black;
+}
+.navigation{
+	justify-content: flex-end;
+	align-items: center;
+	/* background: lightblue; */
+}
+.logo{
+	margin: 0 20px;
+	display: inline-block;
+}
+.w-50{
+	width:50%;
+}
+.button{
+	padding: 4px 50px;
+	border: 1px solid black;
+	color: black;
+}
+.button:hover{
+	background: lightblue;
+	color: white;
+	border: 1px dashed darkblue;
+}
+.text-center{
+	text-align: center;
+}
+/* HOME STYLES */
+.main-header{
+	justify-content: space-around;
+	text-align: center;
+	padding: 80px 0;
+	align-items: center;
+}
+.main-content{
+	padding: 60px 0;
+}
+.features{
+	width: 50%;
+}
+.about{
+	width: 50%;
+	padding-left: 20px;
+}
+.optiontwo{
+	justify-content: space-between;
+}
+.services, .product {
+	padding: 0 20px 0 0;
+}
+.vision{
+	padding: 0 0 20px 0;
+}
+.mid-image{
+	margin: 50px 0px;
+	width: 100%;
+}
+address{
+	margin-top: 50px;
+}
+.phone-number{
+	margin: 20px 0;
+	line-height: 2.5;
+}
+/* SERVICES STYLES */
+.services-header{
+	padding-bottom: 45px;
+}
+.services-header p {
+	line-height: 1.5;
+}
+.services-header-img{
+	margin: 20px 0;
+	width: 100%;
+}
+.services-title {
+	font-size: 2em;
+	margin: 10px 0 25px;
+}
+.service-content{
+	margin: 40px 0;
+	padding: 0 0 40px;
+	flex-wrap: wrap;
+	justify-content: space-between;
+}
+.card{
```

Comment:
Great use of reusable code! This will really benefit you when we switch to React mindset.